README.md

```diff
@@ -118,6 +118,17 @@ Code linting is performed using:
 [Bullet](https://github.com/flyerhzm/bullet) runs around each spec. If it
 detects an N+1 query it will raise an exception and the tests will fail.
 
+### Creating data migrations
+
+When running a live service sometimes you're required to change existing data in
+some way. Using the gem [data_migrate](https://github.com/ilyakatz/data-migrate)
+we can write data migrations in a similar way we do with schema migrations.
+
+The migrations are stored in the `db/data` folder.
+
+When the service deploys it will run these data migrations at the same time as
+the schema ones.
```

Comment:
Can you add instructions on how to generate a migration, and how to run the data migrations at the same time as the DB migrations? (i.e. run `rake db:migrate:with_data`). I know it's in the gem docs, but makes sense to be here too

Reply:
Yeah sure, my worry was that the commands may get out of date as the gem updates but we can cross that bridge when it comes to it 👍